demos/bombs/bombs.cpp

```diff
@@ -3,6 +3,7 @@
 // Purpose:     Bombs game
 // Author:      P. Foggia 1996
 // Modified by: Wlodzimierz Skiba (ABX) since 2003
+// Modified by: Night_Wisp in 2022
```

Comment:
We don't really use those any longer since the switch to Git, you will be credited as the commit author anyhow.